site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig

```diff
@@ -4,6 +4,8 @@
 {% for script in localjs %}
     <script src="{{ script }}"></script>
 {% endfor %}
+<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>  
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.3.2/jspdf.min.js"></script>
```

Comment:
This is probably not how we want to load this library. We could download it locally instead into the libraries folder. Where exactly should the library go?

Reply:
Same as any other dependencies installeable by npm. Add it to package.json, modify .setup/install_submitty/install_site_*.sh